aiven/client/cli.py

```diff
@@ -3997,6 +3997,8 @@ def vpc__user_peer_network_cidr__delete(self) -> None:
         default=False,
         help="Enable termination protection",
     )
+    @arg("--source-service-name", help="Service to fork from")
+    @arg("--recovery-target-time", help="PITR recovery point")
```

Comment:
And also, I'd like to know if it's really necessary to name the command line arg differently from the user_config setting, especially since the help text here uses the exact same words as the setting name. Would `--service-to-fork-from` be wrong?